### ISSUE TITLE
--live-restore enable, but not appearing in docker info

### ISSUE THREAD

**Issue by jgsqware** [moby/moby#34909]
**Description**

I've set the flag `--live-restore` to the `dockerd` but, it doesn't show in the `docker info` command result.

**Steps to reproduce the issue:**
1. Set `--live-restore` in `DOCKER_OPTS` in *drop-ins* systemd unit
2. `systemctl daemon-reload && systemctl restart docker
3. `docker info`

**Describe the results you received:**

1. `systemctl status docker`

```
● docker.service - Docker Application Container Engine
   Loaded: loaded (/usr/lib/systemd/system/docker.service; enabled; vendor preset: disabled)
  Drop-In: /etc/systemd/system/docker.service.d
           └─01-wait-docker.conf, 10-giantswarm-extra-args.conf
   Active: active (running) since Wed 2017-09-20 10:29:54 UTC; 5min ago
     Docs: http://docs.docker.com
 Main PID: 1538 (dockerd)
    Tasks: 37
   Memory: 1.4G
      CPU: 47.704s
   CGroup: /system.slice/docker.service
           ├─1538 dockerd --host=fd:// --containerd=/var/run/docker/libcontainerd/docker-containerd.sock --live-restore --exec-opt native.cgroupdriver=cgroupfs --bip= --selinux-enabled
           └─5380 docker-applyLayer /var/lib/docker/overlay/f4095bbf501dbdfeed1bea871797638de77f6059eaf9091a3aaa977c7848d4d9/tmproot296201535

```

2. `docker info | grep Live-Restore`

*Nothing*

**Describe the results you expected:**

1. `docker info | grep Live-Restore`

```
Live Restore Enabled: true
```

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**

```
Client:
 Version:      1.12.6
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   d5236f0
 Built:        Wed Apr 26 21:47:57 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.12.6
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   d5236f0
 Built:        Wed Apr 26 21:47:57 2017
 OS/Arch:      linux/amd64
```

**Output of `docker info`:**

```
Containers: 11                                                                                                                                                                                
 Running: 10                                                                                                                                                                                  
 Paused: 0                                                                                                                                                                                    
 Stopped: 1                                                                                                                                                                                   
Images: 8                                                                                                                                                                                     
Server Version: 1.12.6                                                                                                                                                                        
Storage Driver: overlay                                                                                                                                                                       
 Backing Filesystem: xfs                                                                                                                                                                      
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: null host bridge overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Security Options: seccomp selinux
Kernel Version: 4.9.24-coreos
Operating System: Container Linux by CoreOS 1353.7.0 (Ladybug)
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 3.679 GiB
Name: xxxxx-aws-stuff
ID: FHB6:BDAK:XYHO:NEEU:4O5F:PP62:YO5G:SE2Q:6DEA:D6KR:3XZJ:O2AT
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Insecure Registries:
 127.0.0.0/8
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
AWS on Container Linux

**Comment by konstruktoid** [comment]
`Server Version: 1.12.6` seems obsolete, tried a version with the new versioning schema?

```
$ docker info | grep -E 'Version|Live'
Server Version: 17.07.0-ce
Kernel Version: 3.10.0-693.2.1.el7.x86_64
Live Restore Enabled: false  
```

```
$ docker info | grep -E 'Version|Live'
Server Version: 17.09.0-ce-rc2
Kernel Version: 4.9.49-moby
Live Restore Enabled: true
```

**Comment by jgsqware** [comment]
Cannot doing that, because we are running k8s on docker and stay on 12.6

**Comment by konstruktoid** [comment]
I believe 12.6 was the last version without this information.

```
$ docker info | grep -E 'Version|Live'
Server Version: 1.13.0
Kernel Version: 3.10.0-693.2.1.el7.x86_64
Live Restore Enabled: true
```

**Comment by thaJeztah** [comment]
Yes, this information was added to `docker info` in this PR; https://github.com/moby/moby/pull/24986, which is included in docker 1.13 and up.

I'm closing this issue, because this is not a bug, and docker 1.12 is no longer maintained, so there's nothing actionable.
